### DISCUSSION TITLE
Can't read moving mesh at corner.

### DISCUSSION THREAD

**Discussion by SangYangSEU** [Anura3D/Anura3D_OpenSource#79] [Q&A] [answered]
I modeled the SHPB(Split Hopkinson Pressure Bar) test with Anura 3D. As shown in figure 1, a gas gun bullet with initial velocity moving to hit the incident bar (third cylinder from left), which later hit the rock sample sandwitched between incident bar and transmitted bar (last cylinder from left).
Actually I don't know how to define moving mesh and compression mesh in this case, cuz when the bullet hit the incident bar, the incident bar almost hit the rock sample at the same time because of high speed (18m/s). I just define the first cylinder's corner points as moving mesh and the other four cylinders as a whole to define compression mesh. Then I got the problem in figure 5 which tell "Can't read moving mesh at corner". Hope I can get help here.
![屏幕截图 2023-03-15 213916](https://user-images.githubusercontent.com/126055850/225328882-c044ab1f-eea2-4635-917a-2e4f989f701d.png)
![屏幕截图 2023-03-15 213837](https://user-images.githubusercontent.com/126055850/225328939-6da799d1-23f5-4c49-85ba-073da21884dd.png)
![屏幕截图 2023-03-15 213955](https://user-images.githubusercontent.com/126055850/225328962-62ee9fca-46d8-4ace-b3cc-f183a9eacfce.png)
![屏幕截图 2023-03-15 214041](https://user-images.githubusercontent.com/126055850/225328971-326c9f5b-864d-432c-8ab7-e22a7103f138.png)
![屏幕截图 2023-03-15 213758](https://user-images.githubusercontent.com/126055850/225332055-4cc00685-14f5-4b64-9b49-3889370e0acd.png)

**Comment by FrancescaCeccato** [comment, marked as answer]
Dear Sang,
did you specify both moving mesh and extending or compressing mesh?
Keep in mind that 

1. the moving mesh follows the movement or a reference material (the bullet?)
2. in your case 4 nodes of the compressing mesh must be on the model boundary (right side) and the other 4 nodes must coincide with the moving mesh; 4 nodes of the moving mesh must be at the boundary (left side).

**Reply by SangYangSEU** [reply]
Thank u Ceccato ！I did't follow the tutorial that compression mesh should be applied on the boundary. Now I have fixed that problem. Really thanks for your help.
